forge-game/src/main/java/forge/game/ability/AbilityUtils.java

```diff
@@ -591,6 +591,15 @@ else if (calcX[0].matches("Enchanted") || calcX[0].matches("Equipped")) {
             }
             val = handlePaid(list, calcX[1], card, ability);
         }
+        else if (calcX[0].matches("CraftedWith")) {
+            final CardCollection list = new CardCollection();
+            for (Card exiled : card.getOwner().getCardsIn(ZoneType.Exile)) {
+                if (exiled.getExiledWith().equals(card)) {
+                    list.add(exiled);
+                }
+            }
+            val = list.isEmpty() ? null :handlePaid(list, calcX[1], card, ability);
```

Comment:
shouldn't just `card.getExiledCards()` be enough?

for correct "linked abilities" can be fixed later

Reply:
Hmm, I'd have to check to make sure that this wouldn't include the card itself in the list too. I think that one would get cleaned up on the zone change back to the battlefield, though so this may work